preprocessors/openstreetmap/osm_service.py

```diff
@@ -941,3 +946,62 @@ def get_coordinates(content):
     """
     if 'coordinates' in content.keys():
         return content['coordinates']
+
+    google_api_key = os.environ["GOOGLE_PLACES_KEY"]
+
+    # Query google places API to find latlong
+    request = f"https://maps.googleapis.com/maps/api/place/details/json?\
+            place_id={content['placeID']}&\
+            key={google_api_key}"
+
+    request = request.replace(" ", "")
+
+    place_response = requests.get(request).json()
+
+    if not check_google_response(place_response):
+        return None
+
+    location = place_response['result']['geometry']['location']
+    coordinates = {
+        'latitude': location['lat'],
+        'longitude': location['lng']
+    }
+
+    return coordinates
+
+
+def check_google_response(place_response):
+    """
+    Helper method to check whether the response from
+    the Google Places API is valid
+
+    Args:
+        place_response: the response from the Google Places API
+
+    Returns:
+        bool: True if valid, False otherwise
+    """
+    if 'result' not in place_response or len(place_response['result']) == 0:
+        logging.error("No results found for placeID")
+        logging.error(place_response)
+        return False
+
+    result = place_response['result']
+
+    if 'geometry' not in result:
+        logging.error("No geometry found for placeID")
+        return False
+
+    if 'location' not in result['geometry']:
+        logging.error("No location found for placeID")
+        return False
+
+    if 'lat' not in result['geometry']['location']:
+        logging.error("No lat found for placeID")
```

Comment:
Recommend just writing out "latitude" and "longitude".

Reply:
Fixed